### ISSUE TITLE
Manage Prestashop install according configuration (.env and environment variables)

### ISSUE THREAD

**Issue by MeKeyCool** [PrestaShop/PrestaShop#28038]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

After #23242 implementation, we can see that configuration usage is not consistent with ``php install-dev/index_cli.php`` command 

### Describe the solution you'd like

if ``.env`` exists or environment variables set, defined values should overwrite default values.

if ``.env`` exists or environment variables set, ``php install-dev/index_cli.php`` should run. So 
- add a --help argument
- run command and don't display help if no argument given

> :question: Should we define some mandatory arguments ? (especially if credentials don't have default value ?)

if arguments are given in command line and ``.env`` exists or environment variables set, cli arguments should be used.

> :warning: Behavior for web install if `.env` exists or environment variables defined should be discussed before implementation. Discussed with @eternoendless and @matks, some credentials may not be displayed during install if given by `.env` file or environment variables. 

> :warning: Some default values should be discussed. Imho, ops and secuirty considerations prefer an error than a default value / behavior that induce thinking everything alright. I think `admin_password``should not have a default value.

> :bulb: Isn't it the right task to refactor `index_cli.php` to a symfony command ? Then we could create 2 tasks : 
> * Refactor `index_cli.php` to a symfony command
> * Refactor web install to use standard PS architecture (symfony controllers, configuration, ...). :thinking: This task may be and epic ... 

### Do you plan to work on this feature?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [x] I'm willing to submit a Pull Request that implements this feature.
- [x] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by hibatallahAouadni** [comment]
Hello @MeKeyCool 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Thanks for your PR!

Thank you

**Comment by MeKeyCool** [comment]
I added some comments to #23242 and I'll probably close this issue soon.

**Comment by hibatallahAouadni** [comment]
Duplicate of #23242
